crates/starknet-types-core/src/felt/num_traits_impl.rs

```diff
@@ -1,6 +1,19 @@
 use super::Felt;
+use num_bigint::{ToBigInt, ToBigUint};
 use num_traits::{FromPrimitive, Inv, One, Pow, ToPrimitive, Zero};
 
+impl ToBigInt for Felt {
+    fn to_bigint(&self) -> Option<num_bigint::BigInt> {
```

Comment:
Are there infallible variants for this trait? It looks like it would be more ergonomic to use given all `Felt`s can be converted.

Reply:
No there is not. This is the way this lib trait is defined so that virtually any type can implement it.

Reply:
Add a comment in the doc to say that it’s safe to unwrap ?

Reply:
done